libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/DSDateRangePicker.test.tsx

```diff
@@ -0,0 +1,184 @@
+    record: mockRecord,
+    formElement: {
+        ...mockFormElementInput,
+        settings: {
+            label,
+            required
+        }
+    },
+    attribute: mockAttributeLink,
+    isReadOnly: false,
+    activeScope: FieldScope.CURRENT,
+    values: {
+        [FieldScope.CURRENT]: {
+            version: null,
+            values: {[idValue]: mockValue}
+        },
+        [FieldScope.INHERITED]: null
+    },
+    metadataEdit: false
+});
+
+describe('DSRangePickerWrapper', () => {
+    const mockOnChange = jest.fn();
+    const mockHandleSubmit = jest.fn();
+    let user!: ReturnType<typeof userEvent.setup>;
+
+    beforeEach(() => {
+        user = userEvent.setup({});
+        jest.resetAllMocks();
+    });
+
+    describe('without required field', () => {
+        test('should call onChange with value', async () => {
+            const state = getInitialState(false);
+            render(
+                <Form>
+                    <Form.Item>
+                        <DSRangePickerWrapper
+                            state={state}
+                            infoButton=""
+                            onChange={mockOnChange}
+                            handleSubmit={mockHandleSubmit}
+                        />
+                    </Form.Item>
+                </Form>
+            );
+
+            const rangePickerInputs = screen.getAllByRole('textbox');
+            await user.click(rangePickerInputs[0]);
+            const currentDate = dayjs().format('YYYY-MM-DD');
+            await user.click(screen.getByTitle(currentDate));
+            await user.click(screen.getByTitle(currentDate));
+
+            expect(mockOnChange).toHaveBeenCalled();
+            expect(mockHandleSubmit).toHaveBeenCalledWith(expect.anything(), state.attribute.id);
+        });
+
+        test('should not save to LEAV if field becomes empty', async () => {
+            const state = getInitialState(false);
+            render(
+                <Form>
+                    <Form.Item>
+                        <DSRangePickerWrapper
+                            state={state}
+                            infoButton=""
+                            onChange={mockOnChange}
+                            handleSubmit={mockHandleSubmit}
+                        />
+                    </Form.Item>
+                </Form>
+            );
+
+            const rangePickerInputs = screen.getAllByRole('textbox');
+            await user.click(rangePickerInputs[0]);
+            const currentDate = dayjs().format('YYYY-MM-DD');
+            await user.click(screen.getByTitle(currentDate));
+            await user.click(screen.getByTitle(currentDate));
+
+            expect(mockOnChange).toHaveBeenCalled();
+            expect(mockHandleSubmit).toHaveBeenCalled();
+
+            await user.click(screen.getByRole('button'));
+
+            expect(mockOnChange).toHaveBeenCalledTimes(2);
+            expect(mockHandleSubmit).toHaveBeenCalledTimes(2);
```

Comment:
**nitpick (blocking):** le titre du test est `should not save to LEAV if field becomes empty`, mais ici on vérifie que c'est appelé 2 fois, il y a une incohérence